.circleci/config.yml

```diff
@@ -19,18 +19,23 @@ jobs:
           command: npm run install-shields
 
       - run:
-          name: Prepare GitHub integration
-          command: |
-            mkdir shields/private
-            echo "{\"gh_token\":\"$GITHUB_TOKEN\"}" > shields/private/secret.json
+          name: Prepare test summary
+          when: always
+          command: mkdir junit
```

Comment:
I bet I added this to the main repo, though I wonder if it's needed. It seems to create at least one level of directory on its own (i.e. we're not creating `junit/app`).

I wonder if we should edit this so it triggers on pushes to PRs, while we're developing it.

Reply:
I was wondering about that too. I know Mocha's xunit reporter will automatically create the directory hierarchy but didn't know about this reporter. 

> I wonder if we should edit this so it triggers on pushes to PRs, while we're developing it.
Definitely!

Reply:
Pretty sure at this point, that this `mkdir` is a no-op 😁